src/index.ts

```diff
@@ -20,49 +21,60 @@ const run = async (): Promise<void> => {
       );
     }
 
-    let prData = await getData(authToken);
+    const coverageConfiguration: PathProperties[] = await getConfiguration();
 
-    if (url && authToken) {
-      try {
-        prData = await sendPullRequestData(url, prData);
-      } catch (error) {
-        console.log(`${error}, Could not send data, printing comment`);
-      }
-    }
+    await Promise.all(
+      coverageConfiguration.map(async (configuration) => {
+        const splitConfigurationPath = configuration.path.split('/');
+        const configurationPath =
+          splitConfigurationPath[splitConfigurationPath.length - 1] === 'lcov.info'
+            ? configuration.path
+            : configuration.path.concat('lcov.info');
+        let prData = await getPullRequestData(configurationPath, configuration.displayName);
```

Comment:
Does this actually change with every different package in the repo?

Reply:
I updated the instructions, so if the person follows them we wouldn't need this logic. But previously the instructions were to not include `/lcov.info` in the path. I still left this part in here in case someone maybe doesn't include `/lcov.info` in their path.
Updated instructions:
```
coverage:
  - displayName: some cool display name
    path: packages/firstpath/coverage/lcov.info
  - displayName: some ... other cool display name
    path: packages/secondpath/coverage/lcov.info
```